shell/browser/api/electron_api_menu_mac.mm

```diff
@@ -145,18 +156,27 @@
     position.x = position.x - [menu size].width;
 
   [popup_controllers_[window_id] setCloseCallback:std::move(close_callback)];
-  // Make sure events can be pumped while the menu is up.
-  base::CurrentThread::ScopedAllowApplicationTasksInNativeNestedLoop allow;
-
-  // One of the events that could be pumped is |window.close()|.
-  // User-initiated event-tracking loops protect against this by
-  // setting flags in -[CrApplication sendEvent:], but since
-  // web-content menus are initiated by IPC message the setup has to
-  // be done manually.
-  base::mac::ScopedSendingEvent sendingEventScoper;
-
-  // Don't emit unresponsive event when showing menu.
-  [menu popUpMenuPositioningItem:item atLocation:position inView:view];
+
+  if (frame) {
+    auto* rfh = frame->render_frame_host()->GetOutermostMainFrameOrEmbedder();
```

Comment:
Probably worth checking this since it can be a nullptr
```suggestion
    auto* frame_rfh = frame->render_frame_host();
    auto* rfh = frame_rfh ? frame_rfh->GetOutermostMainFrameOrEmbedder() : nullptr;
```